space/space.go

```diff
@@ -40,12 +41,15 @@ func (p Space) Equal(u convert.Equaler) bool {
 	if p.Name != other.Name {
 		return false
 	}
+	if p.Description != other.Description {
+		return false
+	}
 	return true
 }
 
 // Repository encapsulate storage & retrieval of spaces
 type Repository interface {
-	Create(ctx context.Context, name string) (*Space, error)
+	Create(ctx context.Context, space Space) (*Space, error)
 	Save(ctx context.Context, space Space) (*Space, error)
```

Comment:
Yes, but `Save(...)` also does the same. I'll change both, rather than have consistency issues.